src/iceflow/data/atm1b.py

```diff
@@ -312,16 +311,14 @@ def extract_itrf(filepath: Path) -> str:
 
     itrf = itrf.upper()
 
-    if itrf not in SUPPORTED_ITRFS:
-        # Try to normalize:
-        try:
-            itrf = {
-                "ITRF05": "ITRF2005",
-                "ITRF08": "ITRF2008",
-            }[itrf]
-        except KeyError as e:
-            err = f"Unrecognized ATM1B ITRF {itrf}"
-            raise RuntimeError(err) from e
+    # Try to normalize based on known ITRF strings in the ATM1B data:
+    try:
+        itrf = {
+            "ITRF05": "ITRF2005",
+            "ITRF08": "ITRF2008",
+        }[itrf]
+    except KeyError:
+        pass
```

Comment:
Can we move this into a Pydantic validator?

Reply:
Perhaps in the future, I'll make a comment. I expect to do some additional refactoring around ITRF normalization to support other dataproducts soon.